docs/quickstart/install-and-use-a-package-in-visual-studio.md

```diff
@@ -1,78 +1,86 @@
+
+## Create a WPF project
 
-You can install the 2019 Community edition for free from [visualstudio.com](https://www.visualstudio.com/) or use the Professional or Enterprise editions.
+You can install a NuGet package in any .NET project, if that package supports the same target framework as the project.
 
-If you're using Visual Studio for Mac, see [Install and use a package in Visual Studio for Mac](install-and-use-a-package-in-visual-studio-mac.md).
+For this quickstart, create a WPF app project:
 
-## Create a project
+1. In Visual Studio, select **File** > **New** > **Project**.
 
-NuGet packages can be installed into any .NET project, provided that the package supports the same target framework as the project.
+1. Enter *.NET* in the search box, select **WPF App (.NET Framework)** for C#, and then select **Next**.
 
-For this walkthrough, use a simple WPF app. Create a project in Visual Studio using **File** > **New Project**, typing **.NET** in the search box, and then selecting the **WPF App (.NET Framework)**. Click **Next**. Accept the default values for **Framework** when prompted.
+1. In the **Configure your new project** window, optionally update the **Project name** and the **Solution name**. Accept the default value for **Framework**, and then select **Create**.
 
-Visual Studio creates the project, which can be found in Solution Explorer. For more information, see [Learn about Solution Explorer](/visualstudio/ide/use-solution-explorer).
+   Visual Studio creates the project, and it appears in [Solution Explorer](/visualstudio/ide/use-solution-explorer).
 
 ## Add the Newtonsoft.Json NuGet package
 
-To install the package, you can use either the NuGet Package Manager or the Package Manager Console. When you install a package, NuGet records the dependency in either your project file or a `packages.config` file (depending on the project format). For more information, see [Package consumption overview and workflow](../consume-packages/Overview-and-Workflow.md).
+To install a NuGet package, you can use either the NuGet Package Manager or the Package Manager Console. Depending on your project format, the installation of a NuGet package records the dependency in either your project file or a *packages.config* file. For more information, see [Package consumption workflow](../consume-packages/overview-and-workflow.md).
 
 ### NuGet Package Manager
 
-1. In Solution Explorer, right-click **References** and choose **Manage NuGet Packages**.
+To use the [NuGet Package Manager](../consume-packages/install-use-packages-visual-studio.md) to install the `Newtonsoft.Json` package in Visual Studio, follow these steps:
 
-    ![Manage NuGet Packages command for project References](media/QS_Use-02-ManageNuGetPackages.png)
+1. In **Solution Explorer**, right-click **References** and choose **Manage NuGet Packages**.
```

Comment:
nit: The menu command is always available on the project node as well. Might be a more natural thing to click there.